bottlecap/src/appsec/processor/context.rs

```diff
@@ -358,14 +509,53 @@ impl Context {
         }
     }
 }
+impl Drop for Context {
+    fn drop(&mut self) {
```

Comment:
Would this mean that there's a possibility the `aws.lambda` span will be completely dropped if appsec is not seen?

Reply:
Yes yes, I am pretty well aware of this... If everything wasn't `async`-tainted, I'd have released the trace in `impl Drop` but I can't, because I can't guarantee I have an async runtime available when `Drop` runs.